Aztec/Classes/TextKit/TextView.swift

```diff
@@ -252,6 +252,10 @@ open class TextView: UITextView {
         font = defaultFont
         
         storage.setHTML(html, withDefaultFontDescriptor: font!.fontDescriptor)
+        if storage.length > 0 && selectedRange.location < storage.length {
+            typingAttributes = storage.attributes(at: selectedRange.location, effectiveRange: nil)
+        }
+        delegate?.textViewDidChangeSelection?(self)
```

Comment:
If this is only for the toolbar, consider calling `TextViewFormattingDelegate`'s `textViewCommandToggledAStyle()` instead.